src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -140,6 +148,8 @@
      */
     public boolean notifyOnItineraryChange = true;
 
+    public ItineraryExistence itineraryExistence;
```

Comment:
This seems like a bit of advance engineering that may or may not end up being used when pausing a monitored trip or determining that it is no longer an option on certain days of the week. I think this is fine for now, but it might get changed a bit later.